setup.py

```diff
@@ -29,8 +30,16 @@ def read_version():
 )
 
 
+class AdminInstall(install):
+    def run(self):
+        self.spawn(["yarn", "install"])
+        self.spawn(["yarn", "build"])
```

Comment:
@webknjaz Any idea how to build with yarn as part of the setup/install process?

The only solutions I'm seeing online, seem to do it like this, which doesn't seem to be doing anything...